2020-01-Winter/0_arrays_sorting/README.md

```diff
@@ -45,19 +45,37 @@ Go to [Solution](#s1)   [Top](#top)
 <!-- Don't remove -->
 <a name="p2"/>
 
-### 2. PROBLEM 2 TODO :bug:
+### 2. Search in Rotated Sorted Array :bug:
```

Comment:
Any text that says "TODO :bug:" can be removed; that text is only a placeholder.

```suggestion
### 2. Search in Rotated Sorted Array
```